msctl

```diff
@@ -1394,7 +1394,19 @@ overviewer() {
   $OVERVIEWER_BIN --config=$SETTINGS_FILE >> $LOG_FILE 2>&1
   $OVERVIEWER_BIN --config=$SETTINGS_FILE --genpoi >> $LOG_FILE 2>&1
 
-  # Add the Google API key to the index.html. 
+  # Add the Google API key to the Overviewer index.html source files.
+  for f in $(find /usr/ -name "index.html" -print | grep overviewer_core/data/web_assets)
```

Comment:
This command returns 2 symbolic links and one file on my server:
```
/usr/share/pyshared/overviewer_core/data/web_assets/index.html
/usr/lib/python2.6/dist-packages/overviewer_core/data/web_assets/index.html
/usr/lib/python2.7/dist-packages/overviewer_core/data/web_assets/index.html
```
Can we just edit the file directly rather than search for it?